src/qt/locale/bitcoin_fr.ts

```diff
@@ -1740,38 +1744,38 @@ p, li { white-space: pre-wrap; }
     </message>
     <message>
         <source>Bytes Sent</source>
-        <translation type="unfinished"></translation>
+        <translation>Octets Envoyés</translation>
     </message>
     <message>
         <source>Bytes Received</source>
-        <translation type="unfinished"></translation>
+        <translation>Octets Reçus</translation>
     </message>
 </context>
 <context>
     <name>PeerTools</name>
     <message>
         <source>Error: Peer-to-peer functionality missing or disabled</source>
-        <translation type="unfinished"></translation>
+        <translation>Erreur : La fonctionnalité Peer-to-peer est absente ou désactivée</translation>
```

Comment:
Peer-to-peer to "Pair-à-pair"

Reply:
I thought about it but in France we all use the English term p2p (peer-to-peer) and not "pair-à-pair".